rules/Strict/Rector/Empty_/DisallowedEmptyRuleFixerRector.php

```diff
@@ -29,7 +29,7 @@ final class DisallowedEmptyRuleFixerRector extends AbstractFalsyScalarRuleFixerR
     public function __construct(
         private readonly ExactCompareFactory $exactCompareFactory,
         private readonly ExprAnalyzer $exprAnalyzer,
-        private readonly UnitializedPropertyAnalyzer $unitializedPropertyAnalyzer
+        private readonly UninitializedPropertyAnalyzer $uninitializedPropertyAnalyzer
```

Comment:
original name is correct

Reply:
Do you mean "Unitialized" is correct?

Reply:
The new name: `UninitializedPropertyAnalyzer` is incorrect, double `nini`

Even new name is correct, please only change comment and tests to avoid BC break

Reply:
> The new name: `UninitializedPropertyAnalyzer` is incorrect, double `nini`

It is Un-[initialized](https://www.merriam-webster.com/dictionary/initialized).

Reply:
I see, I prefer to not change it as it is public service and cause bc break on change